src/api/api.cpp

```diff
@@ -169,6 +169,12 @@ void set_limit_loop_iterations(onnx_options& options, int64_t value)
     options.limit_max_iterations = value;
 }
 
+void set_external_data_path(onnx_options& options,
```

Comment:
https://github.com/ROCm/AMDMIGraphX/blob/eed559feb338b8972335b2aee5d48602d89583b0/tools/api/api.cpp#L157

this API file is auto generated. 

You need to add this `set_external_data_path` function in `tools/api.cpp` instead of directly modifying this src file.

Reply:
Done.